docs/usage/configuration-options.md

```diff
@@ -450,6 +450,71 @@ Make sure that version is a pinned version of course, as otherwise it won't be v
 
 For `sbt` note that Renovate will update the version string only for packages that have the version string in their project's `built.sbt` file.
 
+## bumpVersions
+
+Use this option to configure Renovate to increase a semantic version in your code when it updates dependencies.
+This is useful if Renovate does not support bumping the version in a package file which Renovate does not support, or the version is not in a package file.
+
+For example, if you have a file `.release-version` with the version in it, you can configure Renovate to bump the version in that file to next minor release.
+
+`fileMatch` is a regex pattern to match the file name and follow the same rules.
+
+`matchStrings` is an array of regex patterns to match the version string in the file.
+It must contain a named capture group `version` to capture the version string.
+
+<!-- prettier-ignore -->
+!!! tip
+    You can use templates in `fileMatch`, `bumpType`, and `matchStrings`.
+    This way you can leverage the power of Renovate's templating engine to change based on the context of the upgrade.
+
+```json title="renovate.json with bumpVersions"
+{
+  "bumpVersions": [
+    {
+      "fileMatch": ["\\.release-version"],
+      "bumpType": "minor",
+      "matchStrings": ["^(?<version>.+)$"]
+    }
+  ]
+}
+```
+
+To only bump the version conditionally, you can use [`packageRules`](#packagerules) to select specific upgrades.
+The following example will only bump the version if there has been an upgrade in the `charts/` directory.
+
+```json title="renovate.json with bumpVersions wrapped in a packageRule"
+{
+  "packageRules": [
+    {
+      "matchFileNames": ["charts/**"],
+      "bumpVersions": {
+        "fileMatch": "{{packageFileDir}}\/Chart.ya?ml",
+        "bumpType": "{{#if isPatch}}patch{{else}}minor{{/if}}",
+        "matchStrings": ["version:\\s(?<version>[^\\s]+)"]
+      }
+    }
+  ]
+}
+```
+
+### bumpType
+
+The `bumpType` field can be one of:
+
+- `prerelease`
+- `patch`
+- `minor`
+- `major`
```

Comment:
Is this true? Don't we actually support all release types?:
https://github.com/npm/node-semver/blob/30c438bb46c74f319aa8783f96d233ebf5f4a90d/internal/constants.js#L16-L24

Reply:
As we use the same mechanism as the manager bump mechanism. The assumption is yes. If we want to add additional values this should be in a separate PR for both systems